disconnect-blacklist.json

```diff
@@ -1,8994 +1,8774 @@
-            {
-                "Demand Media": {
-                    "http://www.demandmedia.com/": [
-                        "demandmedia.com",
-                        "indieclick.com"
-                    ]
-                }
-            },
-            {
-                "Delta Projects": {
-                    "http://www.deltaprojects.se/": [
-                        "adaction.se",
-                        "de17a.com",
-                        "deltaprojects.se"
-                    ]
-                }
-            },
-            {
-                "Deutsche Post DHL": {
-                    "http://www.dp-dhl.com/": [
-                        "adcloud.com",
-                        "adcloud.net",
-                        "dp-dhl.com"
-                    ]
-                }
-            },
-            {
-                "Developer Media": {
-                    "http://developermedia.com/": [
-                        "developermedia.com",
-                        "lqcdn.com"
-                    ]
-                }
-            },
-            {
-                "DG": {
-                    "http://www.dgit.com/": [
-                        "dgit.com",
-                        "eyeblaster.com",
-                        "eyewonder.com",
-                        "mdadx.com",
-                        "serving-sys.com",
-                        "unicast.com"
-                    ]
-                }
-            },
-            {
-                "Directtrack": {
-                    "http://directtrack.com/": [
-                        "directtrack.com"
-                    ]
-                }
-            },
-            {
-                "dianomi": {
-                    "http://www.dianomi.com/": [
-                        "dianomi.com"
-                    ]
-                }
-            },
-            {
-                "Didit": {
-                    "http://www.didit.com/": [
-                        "did-it.com",
-                        "didit.com"
-                    ]
-                }
-            },
-            {
-                "Digital River": {
-                    "http://www.digitalriver.com/": [
-                        "digitalriver.com",
-                        "keywordmax.com",
-                        "netflame.cc"
-                    ]
-                }
-            },
-            {
-                "Digital Window": {
-                    "http://www.digitalwindow.com/": [
-                        "digitalwindow.com",
-                        "perfiliate.com"
-                    ]
-                }
-            },
-            {
-                "Digitize": {
-                    "http://www.digitize.ie/": [
-                        "digitize.ie"
-                    ]
-                }
-            },
-            {
-                "Direct Response Group": {
-                    "http://www.directresponsegroup.com/": [
-                        "directresponsegroup.com",
-                 
```

Comment:
There are a bunch of new Google properties domains here which aren't in the entitylist.